file-uploader.js

```diff
@@ -46,7 +46,7 @@ export function fileUploader(submissionBody, submissionFormat, requestHeaders, b
     }, requestHeaders)
   };
 
-  return fileUploaderUtil.validateSubmission(submissionBody, submissionFormat, baseOptions)
+  return fileUploaderUtil.validateSubmission(JSON.parse(submissionBody), submissionFormat, baseOptions)
```

Comment:
Where does the `submissionBody` come from? Does it need to be parsed here?

Reply:
This is an argument passed in from a user of this package. The README states that this will be a JSON string. If I do not parse this here, the validate submission method from the current API returns a JSON parse error "Unexpected token " at position 0".

Reply:
@petemckinneysb let's back this change out if we found that it is working as expected now, that way axios should be able to handle if it is a json string, or an object. If we call parse and it is not a string though that could cause a bug.